source/cloudstack-faq.html.markdown

```diff
@@ -1,58 +0,0 @@
----
-title: Apache CloudStack FAQ
----
-
-<div class="row">
-
-<div class="col-lg-12">
-
-<div class="page-header">
-
-<h1 id="indicators">Apache CloudStack: FAQ</h1>
-
-</div>
-
-</div>
-
-</div>
-
-This page is for Frequently Asked Questions (FAQ) having to do with Apache CloudStack as a project and its use.
```

Comment:
Removing this would cause 404 on http://cloudstack.apache.org/cloudstack-faq.html @PaulAngus ?

Reply:
Have I missed a link to the page somewhere?  I've removed it from the menu.
IMO the page would actually detract from someone's view of CloudStack as it appears to be half started.